scripts/create_repository.py

```diff
@@ -0,0 +1,198 @@
+    return "io_bazel_rules_scala_" + coordinate.artifact.split('_')[0].replace('-', '_')
+  elif "org.openjdk.jmh" in coordinate.group or "org.ow2.asm" in coordinate.group or "net.sf.jopt-simple" in coordinate.group or "org.apache.commons" in coordinate.group or "junit" in coordinate.group or "org.hamcrest" in coordinate.group or "org.specs2" in coordinate.group:
+    return "io_bazel_rules_scala_" + coordinate.group.replace('.', '_').replace('-', '_') + '_' + coordinate.artifact.split('_')[0].replace('-', '_')
+  elif "mustache" in coordinate.group or "guava" in coordinate.group or "scopt" in coordinate.group:
+    return "io_bazel_rules_scala_" + coordinate.group.split('.')[-1]
+  elif "com.thesamet.scalapb" in coordinate.group or "io." in coordinate.group or "com.google.guava" in coordinate.group:
+    return "scala_proto_rules_" + coordinate.artifact.split('_')[0].replace('-', '_')
+  else:
+    return (coordinate.group.replace('.', '_').replace('-', '_') + '_' + coordinate.artifact.split('_')[0].replace('-', '_')).replace('_v2', '')
+
+def map_to_resolved_artifacts(output) -> List[ResolvedArtifact]:
+  resolved_artifacts = []
+  subprocess.call(f'cs fetch {' '.join(output)} --json-output-file out.json', shell=True)
+  for o in output:
+    replaced = o.replace(':default','')
+    coordinates = get_maven_coordinates(replaced)
+    checksum = get_artifact_checksum(replaced)
+    direct_dependencies = get_json_dependencies(replaced)
+    resolved_artifacts.append(ResolvedArtifact(coordinates, checksum, direct_dependencies))
+  return resolved_artifacts
+
+def resolve_artifacts_with_checksums_and_direct_dependencies(root_artifacts) -> List[ResolvedArtifact]:
+  command = f'cs resolve {' '.join(root_artifacts)}'
+  output = subprocess.run(command, capture_output=True, text=True, shell=True).stdout.splitlines()
+  return map_to_resolved_artifacts(output)
+
+def to_rules_scala_compatible_dict(artifacts, version) -> Dict[str, Dict]:
+  temp = {}
+
+  for a in artifacts:
+    label = get_label(a.coordinates).replace('scala3_', 'scala_').replace('scala_tasty_core', 'scala_scala_tasty_core')
+    deps = [f'@{get_label(dep)}_2' if "scala3-library_3" in a.coordinates.artifact else f'@{get_label(dep)}' for dep in a.direct_dependencies]
+
+    temp[label] = {
+        "artifact": f"{a.coordinates.coordinate}",
+        "sha256": f"{a.checksum}",
+    } if not deps else {
+        "artifact": f"{a.coordinates.coordinate}",
+        "sha256": f"{a.checksum}",
+        "deps:": deps,
+      }
+
+  return temp
+
+def is_that_trailing_coma(content, char, indice) -> bool:
+  return content[indice] == char and content[indice+1] != ',' and content[indice+1] != ':' and content[indice+1] != '@' and not content[indice+1].isalnum()
+
+def get_with_trailing_commas(content) -> str:
```

Comment:
After mapping to the dict, there are no trailing commas, so this is a workaround to adjust file to .bzl files / starlark convention and CI-checks failures.